jsx_test.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import Card from "./card";
+
+export default class CardList extends React.Component {
+  constructor(props)
+  {
+    super(props);
+  }
+
+  render () {
+    var card_list = this.props.deck_list.map(function(card)
+      {
+        return(
+          <Card card={card} />
+        );
+      });
```

Comment:
Expected indentation of 4 space characters but found 6                         indent<br>Extra semicolon                                                                semi